src/mame/roland/roland_d70.cpp

```diff
@@ -510,13 +517,13 @@ void roland_d70_state::descramble_rom_internal(u8 *dst, const u8 *src) {
 
 ROM_START(d70)
 	ROM_REGION(0x20000, "maincpu", 0)
-	ROM_DEFAULT_BIOS("v1.19")
-	ROM_SYSTEM_BIOS( 0, "v1.19", "Version 1.19 - March 9, 1993" )
-	ROM_SYSTEM_BIOS( 1, "v1.16", "Version 1.16 - January 28, 1991" )
-	ROM_SYSTEM_BIOS( 2, "v1.14", "Version 1.14 - September 20, 1990" )
-	ROM_SYSTEM_BIOS( 3, "v1.12", "Version 1.12 - August 8, 1990" )
-	ROM_SYSTEM_BIOS( 4, "v1.10", "Version 1.10 - April 19, 1990" )
-	ROM_SYSTEM_BIOS( 5, "v1.00", "Version 1.00 - March 10, 1990" )
+	ROM_DEFAULT_BIOS("v119")
+	ROM_SYSTEM_BIOS( 0, "v119", "Version 1.19 - March 9, 1993" )
+	ROM_SYSTEM_BIOS( 1, "v116", "Version 1.16 - January 28, 1991" )
+	ROM_SYSTEM_BIOS( 2, "v114", "Version 1.14 - September 20, 1990" )
+	ROM_SYSTEM_BIOS( 3, "v112", "Version 1.12 - August 8, 1990" )
+	ROM_SYSTEM_BIOS( 4, "v110", "Version 1.10 - April 19, 1990" )
+	ROM_SYSTEM_BIOS( 5, "v100", "Version 1.00 - March 10, 1990" )
```

Comment:
What's the reasoning for omitting the dots?

How does one deal with the ambiguity when some ROM eventually reaches version `v10.0`?

Reply:
It’s unnecessary clutter when the minor version is always two digits and there’s no patch version.  If they went to two-digit major versions, it would still be obvious that `v1000` is `10.00`.

For other cases, see e.g. this (major version and minor version always one digit, patch version may or may not be present):
https://github.com/mamedev/mame/blob/mame0273/src/mame/osborne/osborne1.cpp#L1069

Or this:
https://github.com/mamedev/mame/blob/mame0273/src/mame/acorn/z88.cpp#L609

Or this:
https://github.com/mamedev/mame/blob/mame0273/src/mame/alesis/alesis.cpp#L480

We don’t put punctuation in the BIOS short names unless it’s really necessary for it to work as a mnemonic.